index.js

```diff
@@ -80,7 +111,7 @@ function lint(source, options, webpack, done) {
         emitter = webpack.emitWarning;
       }
 
-      emitter(options.formatter(report));
+      emitter(new Error(options.formatter(report)));
```

Comment:
I believe the tests are failing because this is now being emitted as an error. This shouldn't be the case, especially if emitting as a warning not an error.